SlowCheetah.VisualStudio/SlowCheetahPackage.cs

```diff
@@ -432,11 +458,11 @@ private bool TryGetFileToTransform(IVsHierarchy hierarchy, uint parentId, string
                 }
                 else
                 {
-                    while (childId != VSConstants.VSITEMID_NIL)
+                    while (docId != VSConstants.VSITEMID_NIL)
                     {
-                        hierarchy.GetProperty(childId, (int)__VSHPROPID.VSHPROPID_NextVisibleSibling, out childIdObj);
-                        childId = (uint)(int)childIdObj;
-                        if (ErrorHandler.Succeeded(project.GetMkDocument(childId, out documentPath)))
+                        hierarchy.GetProperty(docId, (int)__VSHPROPID.VSHPROPID_NextVisibleSibling, out childIdObj);
+                        docId = (uint)(int)childIdObj;
+                        if (ErrorHandler.Succeeded(project.GetMkDocument(docId, out documentPath)))
                         {
                             if (PackageUtilities.IsFileTransform(Path.GetFileName(documentPath), transformName, configs))
```

Comment:
I guess `documentPath` can be potentially non-null in a return false scenario. Wherever you have a `return false` I would explicitly set `docId` and `documentPath` to their default values just to be safe.